game_eggs/gta/fivem/egg-five-m.json

```diff
@@ -60,24 +60,14 @@
         },
         {
             "name": "fivem version",
-            "description": "The fivem version that is to be installed. Invalid versions will default to recommended.\r\n\r\nAn example is `1383-e5ea040353ce1b8bc86e37982bf5d888938e3096`\r\n\r\nYou can get the latest version from here - https:\/\/runtime.fivem.net\/artifacts\/fivem\/build_proot_linux\/master\/",
+            "description": "The fivem version that is to be installed. Invalid versions will default to recommended.\r\n\r\nAn example is `6007-f9c407539e79a33772db3cbf47ab8d5e9d57e7a4`\r\n\r\nYou can get the latest version from here - https:\/\/runtime.fivem.net\/artifacts\/fivem\/build_proot_linux\/master\/",
             "env_variable": "FIVEM_VERSION",
             "default_value": "recommended",
             "user_viewable": true,
             "user_editable": true,
             "rules": "required|string|max:50",
             "field_type": "text"
         },
-        {
-            "name": "Download Link",
```

Comment:
I would rather keep this if someone ones to host their own copy of the server files internally or something

Reply:
from Quintes Changelog:

`remove the DOWNLOAD_URL variable as now the FIVEM_VERSION variable accepts versions`

Reply:
The fivem version variable can be a full version string and it will download it. If you refere that I also add this option no problem to me